GOVERNANCE.md

```diff
@@ -1,137 +1,129 @@
 modification via pull request and it will be considered by the project
-Collaborators. All pull requests must be reviewed and accepted by a
-Collaborator with sufficient expertise who is able to take full
+Members. All pull requests must be reviewed and accepted by a
+Member with sufficient expertise who is able to take full
 responsibility for the change. In the case of pull requests proposed
-by an existing Collaborator, an additional Collaborator is required
+by an existing Member, an additional Member is required
 for sign-off.
 
-If one or more Collaborators oppose a proposed change, then the change can not
+If one or more Members oppose a proposed change, then the change can not
 be accepted unless:
 
-* Discussions and/or additional changes result in no Collaborators objecting to
-  the change. Previously-objecting Collaborators do not necessarily have to
+* Discussions and/or additional changes result in no Members objecting to
+  the change. Previously-objecting Members do not necessarily have to
   sign-off on the change, but they should not be opposed to it.
-* The change is escalated to the TSC and the TSC votes to approve the change.
-  This should only happen if disagreements between Collaborators cannot be
+* The change is escalated to the TC and the TC votes to approve the change.
+  This should only happen if disagreements between Members cannot be
   resolved through discussion.
 
-Collaborators may opt to elevate significant or controversial modifications to
-the TSC by assigning the `tsc-review` label to a pull request or issue. The
-TSC should serve as the final arbiter where required.
+Members may opt to elevate significant or controversial modifications to
+the TC by assigning the `tc-review` label to a pull request or issue. The
+TC should serve as the final arbiter where required.
 
-* [Current list of Collaborators](./README.md#current-project-team-members)
-* [A guide for Collaborators](./COLLABORATOR_GUIDE.md)
+* [Current list of Members](./README.md#current-project-team-members)
+* [A guide for Members](./COLLABORATOR_GUIDE.md)
 
-### Collaborator Activities
+### Member Activities
 
-Typical activities of a Collaborator include:
+Typical activities of a Member include:
 
 * helping users and novice contributors
 * contributing code and documentation changes that improve the project
 * reviewing and commenting on issues and pull requests
-* participation in working groups
+* participation in adjacent projects
 * merging pull requests
 
-The TSC periodically reviews the Collaborator list to identify inactive
-Collaborators. Past Collaborators are typically given _Emeritus_ status. Emeriti
-may request that the TSC restore them to active status.
+The TC periodically reviews the Member list to identify inactive
+Members. Past Members are typically given _Emeritus_ status. Emeriti
+may request that the TC restore them to active status.
 
-## Technical Steering Committee
+## Technical Committee
```

Comment:
Is how TC members are added documented anywhere (or will they be)? Will the TC have its own repo like Node.js TSC/CTC does?

Reply:
it will be, yeah!